ha/core/event_manager/event_manager.py

```diff
@@ -332,4 +345,16 @@ def message_type(self, component: str) -> str:
         Args:
             component (str): component name.
         """
+        key = const.EVENT_MGR_MESSAGE_TYPE_KEY.replace("<component_id>", component)
+        value = None
+        Log.debug(f"Fetching message type for {key}")
+
```

Comment:
kv = self._confstore.get(key)
if kv is None:
      return kv
else:
      ....

Reply:
Done